### DISCUSSION TITLE
Fehler beim Update von node-red-contrib-philipshue-events

### DISCUSSION THREAD

**Discussion by PvtOta** [andesse/hue-clip-api.node-red-flows#39] [General]
Hallo zusammen,

NodeRed hat mir angezeigt, dass @yadomi/node-red-contrib-philipshue-events aktualisiert werden kann. Nach dem Update habe ich NodeRed neu gestartet. Danach waren die Nodes allerdings nicht mehr installiert.

Die Neuinstallation scheitert leider ebenfalls. Es erscheint die folgende Meldung:
![image](https://user-images.githubusercontent.com/14905229/230740494-281dc458-e6b7-40b1-ab61-e4cc48f8ad13.png)

Hat jemand einen Tipp, wie ich das noch retten kann?

Danke vorab.
Viele Grüße

**Comment by andesse** [comment]
Hi @PvtOta, da kann ich leider nichts zu sagen. Da musst Du bei Yadomi nen Ticket aufmachen:)
Nimm einfach den SSE Event Stream. https://github.com/andesse/HUE-CLIP-API.Node-RED-Flows/blob/main/flows/SSE_Event_Stream.json

der basiert auf der sehr einfachen contrib node-red-contrib-sse-client, bitte vorher installieren 
Du kannst den key von der Yadomi config nehmen. Wenn du Bewegungsmelder hast, bleibt der stream ewig am Leben.

Gruß 
Andreas

**Reply by PvtOta** [reply]
Vielen Dank für die super-schnelle Antwort!

**Comment by andesse** [comment]
Bei Rückfragen nochmal melden
